### PR TITLE
Add logs

### PR DESCRIPTION
**Pull request by synthorchid**
ref https://github.com/ethercrow/lightstep-haskell/issues/4

### CODE REVIEW

File: src/SomeProject.hs [hunk 1]
```diff
@@ -1,7 +0,0 @@
-module SomeProject
```
Comment (ethercrow) [inline src/SomeProject.hs]: 👍

File: src/LightStep/HighLevel/IO.hs [hunk 2]
```diff
@@ -28,6 +28,15 @@ import System.Timeout
 globalSharedMutableSpanStacks :: MVar (HM.HashMap ThreadId [Span])
 globalSharedMutableSpanStacks = unsafePerformIO (newMVar mempty)
 
+data LogEntryKey =  ErrorKind | Event | Message | Stack | Custom String
```
Comment (ethercrow) [inline src/LightStep/HighLevel/IO.hs]: Why not `Custom T.Text` since it must be converted to Text anyway?

Comment (synthorchid) [reply src/LightStep/HighLevel/IO.hs]: Indeed, changed!

**Comment by ethercrow** [comment]
Let's call it `addLog` or `writeLog` instead of `setLog`? It would better reflect that `addLog foo bar` doesn't overwrite `addLog foo baz` unlike `setTag foo bar`.

**Comment by synthorchid** [comment]
@ethercrow sures, changed to `addLog`.

**Comment by ethercrow** [comment]
Thanks!

**Pull request merged**